dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -203,6 +224,15 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
         return *this;
     }
 
+    winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationBuilder AppNotificationBuilder::AddComboBox(AppNotificationComboBox const& value)
+    {
+        ThrowIfMaxInputItemsExceeded();
+
+        m_comboBoxList.push_back(value);
+
+        return *this;
+    }
+
     winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationBuilder AppNotificationBuilder::SetTag(hstring const& value)
     {
         m_tag = value;
```

Comment:
@pmpurifoy I think we need to make sure all these Set*(Hstring) APIs don't take in empty strings. They can be problematic. At the same time, m_tag and m_group should default to empty if they havent been set already

Reply:
Are empty string invalid in the xml schema? The goal for the builder at this point is to produce valid xml and keep the validation to the moinimum required to do so.

there's a deliverable that captures the plans to increase validation: https://microsoft.visualstudio.com/OS/_workitems/edit/40746502